pom.xml

```diff
@@ -92,6 +88,8 @@
                 <includes>
                   <include>com.datastax.oss:java-driver-shaded-guava</include>
                   <include>com.google.guava:guava</include>
+                  <include>com.google.guava:failureaccess</include>
+                  <include>com.google.j2objc:j2objc-annotations</include>
```

Comment:
Since 26.0 compiling code that depends on `java-driver-shaded-guava` fails with a warning:
```
integration-tests/src/test/java/com/datastax/oss/driver/mapper/GuavaFutureProducerService.java:[-1,-1] unknown enum constant com.datastax.oss.driver.shaded.guava.j2objc.annotations.ReflectionSupport.Level.FULL
  reason: class file for com.datastax.oss.driver.shaded.guava.j2objc.annotations.ReflectionSupport$Level not found
```
I couldn't figure this out, even after reading through the list of changes from `25.1` -> `26.0` but including this jar in the shaded jar fixes the issues